tests/puppeteer/test/smoke/checkFO.js

```diff
@@ -0,0 +1,43 @@
+// Importing pages
+const FO_PAGE = require('../../pages/FO/FO_home');
+const FO_PRODUCT_Page = require('../../pages/FO/FO_product');
+const FO_CART_Page = require('../../pages/FO/FO_cart');
+
+let page;
+let FO_HOME;
+let FO_PRODUCT;
+let FO_CART;
+
+let productData = {
+  name: 'T-SHIRT IMPRIMÉ COLIBRI',
+  quantity: "1",
+  price: '22.94',
+  description: 'Symbole de légèreté et de délicatesse',
+};
```

Comment:
We need to put that data in a separate file, in cas we need to reuse it elsewhere or we want to use another language.

Reply:
Yess ;) thank you